### ISSUE TITLE
When you do synchronize as true, it will try to alter the database tables with the structure created in model files.

### ISSUE THREAD

**Issue by Saisreenivas** [bhaveet/odin#1]
Keep synchronize as true only if the database structure should be altered based on model files in the repository.

https://github.com/bhaveet/odin/blob/e5fc50745955fa1f92e6b956781b6ca1a1faa850/src/connector/connections/Postgresql.ts#L18

**Comment by bhaveet** [comment]
True. That have been added just for testing(experimentation) locally. You can see I am importing configs above.
